src/features/stats/index.js

```diff
@@ -52,6 +77,14 @@ class Stats extends Component {
         else if (hpPercent >= 97) hpBorder = '5px 3px 3px 5px';
         else hpBorder = '5px 0 0 5px';
 
+        let manaPercent = (mana / maxMana) * 100;
+        if (manaPercent > 100) manaPercent = 100;
+
+        let manaBorder;
+        if (manaPercent >= 99) manaBorder = '5px';
+        else if (manaPercent >= 97) manaBorder = '5px 3px 3px 5px';
+        else manaBorder = '5px 0 0 5px';
```

Comment:
Instead of the `else` clause, why not initialise `manaBorder` with that value?

Reply:
It's how the hpBorder is calculated above, and it practically makes no difference which one is used. Personally, I kind of prefer it this way.